src/Components/Contentlet.js

```diff
@@ -2,22 +2,17 @@ import React, { Component } from 'react';
 import Loadable from 'react-loadable';
 
 import { Spinner } from 'reactstrap';
-
 import DotContentlet from '../libs/DotContentlet';
-import CantRender from './CantRender';
-
+import SimpleWidget from './DotCMS/SimpleWidget';
 import PageContext from '../PageContext';
 
 const Loading = (props) => {
     const divStyle = {
         textAlign: 'center',
         padding: '2rem'
     };
-
     return props.error ? (
-        <CantRender color="warning" title={props.contentlet.title}>
-            <p>{props.error.message}</p>
-        </CantRender>
+        <SimpleWidget identifier={props.contentlet.identifier} />
```

Comment:
We can't do this because this will fallback also for contentlets, we need the `<SimpleWidget />` only for widget, contentlets that doesn't have React Components should show the warn (at least for now).